lib/models/zalo_profile_model.dart

```diff
@@ -34,32 +36,34 @@ class ZaloProfileModel {
         picture: picture ?? this.picture,
       );
 
-  factory ZaloProfileModel.fromJson(Map<dynamic, dynamic> json) => ZaloProfileModel(
+  factory ZaloProfileModel.fromJson(Map<dynamic, dynamic> json) =>
+      ZaloProfileModel(
         birthday: json["birthday"] == null ? null : json["birthday"],
         gender: json["gender"] == null ? null : json["gender"],
         name: json["name"] == null ? null : json["name"],
         id: json["id"] == null ? null : json["id"].toString(),
-        picture: json["picture"] == null ? null : Picture.fromJson(json["picture"]),
+        picture:
+            json["picture"] == null ? null : Picture.fromJson(json["picture"]),
       );
```

Comment:
**Suggestion:** Use explicit type instead of dynamic for better type safety in fromJson method.
```suggestion
  factory ZaloProfileModel.fromJson(Map<String, dynamic> json) =>
        ZaloProfileModel(
          birthday: json["birthday"] == null ? null : json["birthday"],
          gender: json["gender"] == null ? null : json["gender"],
          name: json["name"] == null ? null : json["name"],
          id: json["id"] == null ? null : json["id"].toString(),
          picture: json["picture"] == null ? null : Picture.fromJson(json["picture"]),
        );
```